src/test/resources/logging.properties

```diff
@@ -0,0 +1,3 @@
+.level=FINEST
+handlers=java.util.logging.ConsoleHandler
+java.util.logging.ConsoleHandler.level=FINEST
```

Comment:
add empty line at the end of the files

Reply:
Regarding to `what about this ? ` feedback: 
Simplified the  `shouldDeleteDirectory()` and 'shouldDelete()'  methods using inline code, here is the commit: 
https://github.com/jenkinsci/build-history-manager-plugin/pull/84/commits/8c589079a39e5facf88d3fb3291f63cf907e624c